.github/workflows/add-to-project.yml

```diff
@@ -5,13 +5,23 @@ on:
     types:
       - opened
       - transferred
+env:
+  GH_APP_ID_DISTRO_CI: ${{ secrets.GH_APP_ID_DISTRO_CI }}
```

Comment:
Since these secrets are not used more than once, I'd say no need for this env part and let's use the secrets directly.
No much added value from `secrets.GH_APP_ID_DISTRO_CI` to `env.GH_APP_PRIVATE_KEY_DISTRO_CI`.

Reply:
reverted back to original with no vars.